### DISCUSSION TITLE
Run Uno.QuickStart/MyApp.Skia.Linux.FrameBuffer in ubuntu 20.04 issue

### DISCUSSION THREAD

**Discussion by zxslzx** [unoplatform/uno#7828] [Q&A]
Segmentation fault occurred
![image](https://user-images.githubusercontent.com/32009362/149618191-cf51ef54-48c2-4e2f-a7e4-e50d90409e35.png)

**Comment by jeromelaban** [comment]
Thanks for the report. Do you have a repro case we can use ?
